README.md

```diff
@@ -5,6 +5,12 @@ The `TDS_ConfigService` (aka Config Support Service) consists of two modules:
 * **client:** Contains the POJOs/classes needed for a consumer to interact with the Config Support Service
 * **service:** REST endpoints that provide TDS configuration data
 
+## Prerequisites
```

Comment:
@jeffjohnson9046 I kinda feel like this project's documentation is the most complete.  Do you think I should carry this forward to the other projects?  Specifically the docker portion and the migration file explanation.